### ISSUE TITLE
YouTube link Action Bar Button doesn't show just ends up in the Menu

### ISSUE THREAD

**Issue by Jonnyblue** [Jonnyblue/PuertoGaleraDiveSites#1]
I've been struggeling with getting an action button on my DivesiteFragment.java fragment Activity.

Here is the menu.xml, everything I've googled seems to say that this is correct.

<code> 
<menu xmlns:android="http://schemas.android.com/apk/res/android"
      xmlns:puertogaleradivesites="http://schemas.android.com/apk/res-auto" >
    
    <item android:id="@+id/youtube"
          android:icon="@drawable/ic_play_arrow_black_24dp"
          android:title="YouTube"
          puertogaleradivesites:showAsAction="ifRoom"/>
    <item android:id="@+id/action_settings"
          puertogaleradivesites:showAsAction="never"
          android:title="Settings"/>
    ...
</menu></code>

And here is the DivesiteFragment.java

<code>
package com.blueribbondivers.puertogaleradivesites;

import android.content.Context;
import android.content.res.Resources;
import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.text.method.ScrollingMovementMethod;
import android.view.LayoutInflater;
import android.view.Menu;
import android.view.MenuInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import java.util.UUID;

/**
- Created by jonathan on 24/06/15.
  */
  public class DivesiteFragment extends Fragment {
  private Divesite mDivesite;
  private TextView mDepthField;
  private TextView mSiteDescription;
  private TextView mDistance;
  private ProportionalImageView mImageView;
  public static final String EXTRA_SITE_ID = "com.blueribbondivers.extraSiteID";
  private Context mContext;
  
  @Override
  public void onCreate(Bundle savedInstanceState){
      super.onCreate(savedInstanceState);
      UUID crimeId = (UUID)getActivity().getIntent()
              .getSerializableExtra(EXTRA_SITE_ID);
      mDivesite = DiveSites.get(getActivity()).getDiveSite(crimeId);
      mContext = getActivity().getApplicationContext();
      //getActivity().getActionBar().setTitle(Html.fromHtml("<font color=\"#1c3565\">" + mDivesite.getName() + "</font>"));
  
  }
  
  @Override
  public View onCreateView(LayoutInflater inflater, ViewGroup parent, Bundle savedInstanceState) {
      View v = inflater.inflate(R.layout.fragment_divesite, parent, false);
      Resources resources = mContext.getResources();
      //setHasOptionsMenu(true);
      mDepthField = (TextView)v.findViewById(R.id.divesite_display_depth);
      mDepthField.setText(mDivesite.getMaxDepth());
      mSiteDescription = (TextView)v.findViewById(R.id.divesite_display_description);
      mSiteDescription.setText(mDivesite.getSiteDescription());
      mSiteDescription.setMovementMethod(new ScrollingMovementMethod());
      mDistance = (TextView)v.findViewById(R.id.divesite_display_distance);
      mDistance.setText(mDivesite.getTravelTime());
      mImageView = (ProportionalImageView)v.findViewById(R.id.divesite_display_imageView);
      String imageName = mDivesite.getPhoto();
      final int resourceID = resources.getIdentifier(imageName,"drawable",mContext.getPackageName());
      mImageView.setImageResource(resourceID);
      setHasOptionsMenu(true);
  
  ```
  return v;
  ```
  
  }
  
  @Override
  public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
      inflater.inflate(R.menu.divesitemenu, menu);
      super.onCreateOptionsMenu(menu,inflater);
  }
  }
  </code>

Any help would be great.

**Comment by Ajeet-Meena** [comment]
Are you inflating the right menu ?

**Comment by Jonnyblue** [comment]
Yeah, sorry missed that bit in code. All items added to the menu end up in the overflow menu. Nothing ever ends up on the bar.

**Comment by Ajeet-Meena** [comment]
Hello, I tried cloning your project. The manifest file were missing from project.
